c-bindings-gen/src/types.rs

```diff
@@ -1788,24 +1788,28 @@ impl<'a, 'c: 'a> TypeResolver<'a, 'c> {
 		self.write_to_c_conversion_inline_suffix_inner(w, t, generics, false, ptr_for_ref, false);
 	}
 
-	fn write_from_c_conversion_prefix_inner<W: std::io::Write>(&self, w: &mut W, t: &syn::Type, generics: Option<&GenericTypes>, is_ref: bool, ptr_for_ref: bool) {
+	fn write_from_c_conversion_prefix_inner<W: std::io::Write>(&self, w: &mut W, t: &syn::Type, generics: Option<&GenericTypes>, is_ref: bool, _ptr_for_ref: bool) {
 		self.write_conversion_inline_intern(w, t, generics, is_ref, false, false, "() /*", true, |_, _| "&local_".to_owned(),
 				|a, b, _c| self.from_c_conversion_prefix_from_path(a, b),
-				|w, decl_type, _full_path, is_ref, is_mut| match decl_type {
-					DeclType::StructImported if is_ref && ptr_for_ref => write!(w, "unsafe {{ &*(*").unwrap(),
-					DeclType::StructImported if is_mut && is_ref => write!(w, "unsafe {{ &mut *").unwrap(),
-					DeclType::StructImported if is_ref => write!(w, "unsafe {{ &*").unwrap(),
-					DeclType::StructImported if !is_ref => write!(w, "*unsafe {{ Box::from_raw(").unwrap(),
-					DeclType::MirroredEnum if is_ref => write!(w, "&").unwrap(),
-					DeclType::MirroredEnum => {},
+				|w, decl_type, _full_path, is_ref, _is_mut| match decl_type {
+					DeclType::MirroredEnum if is_ref => write!(w, "{{ use crate::c_types::mapping::*; &").unwrap(),
```

Comment:
We should just add the use to the top of all files in the default use string.

Reply:
Yeah I was also planning to do that, this is just a temporary thing